src/wrapper/gl.zig

```diff
@@ -91,7 +102,7 @@ pub const Attribute = union(AttributeName) {
     blue_size: usize,
     alpha_size: usize,
     buffer_size: usize,
-    doublebuffer: usize,
+    doublebuffer: bool,
```

Comment:
Is that truly a `bool` tho? or is the C field a `usize`?

Reply:
The docs suggest that double buffering is set to ON or OFF